src/Tabs/Tab.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, HTMLAttributes, useCallback } from 'react';
+import styled, { css } from 'styled-components';
+import { TabContext, ContextProps } from './Tabs';
+
+const TabComponent = styled.div`
+  margin-right: 39px;
+  display: flex; 
+  align-items: center;  
+`;
+
+const TabLabel = styled.label<{ active: boolean }>`
+  height: 40px;
+  ${({theme}) => css`
+    font-family: ${theme.fontFamily.ui};
+  `}
+  font-size: 15px;
+  font-weight: ${({ active }) => active ? '600' : '500'};
+  letter-spacing: 0.09px;
+  color: ${({ active }) => active ? '#0097ce' : '#787878'};
+  cursor: pointer;
+  border-bottom: ${({ active }) => active ? '2px solid #0097ce' : '2px solid transparent'};
+  padding-top: 6px;
+  padding-bottom: 14px;
+`;
+
+interface OwnProps {
+  tabFor: string
+}
+
+type Props = OwnProps & HTMLAttributes<HTMLDivElement>
+
+const Tab: React.FC<Props> = ({ children, tabFor }) => {
```

Comment:
Maybe `, ...props` here and below on TabComponent div

Reply:
Done